core/pom.xml

```diff
@@ -8,6 +8,7 @@
     </parent>
 
     <artifactId>cucumber-core</artifactId>
+    <version>4.0.2-SNAPSHOT</version>
```

Comment:
Please rebase on master instead. This will cause merge conflicts down the line. You can ignore the failing semantic versioning check at the moment.